.github/workflows/pypi.yml

```diff
@@ -0,0 +1,81 @@
+on:
+  push:
+    tags:
+      - 'v[0-9]*'
```

Comment:
Should this be `v[0-9]+` (one or more digit), instead of having the digit be optional?

Reply:
This doesn't match `vasdasd`: github's syntax is more like globs than regexes. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
our tags follow semver convention (more or less), so I think matching one or more digit is fine

Reply:
Somehow didn't realize that this is possible, at least according to the docs I linked:

```suggestion
      - 'v[0-9]+'
```

Reply:
Thanks for the quick response many moons later - and the work!